website/src/scripts/setupSolidityDocs.ts

```diff
@@ -0,0 +1,71 @@
+import fs from "fs";
+import path from "path";
+
+const solidityDocDir = path.resolve(__dirname, "../../versioned_docs/version-v1.x/natspec-docs");
```

Comment:
I'd suggest maybe `solidity-api` or `contracts-api` instead of `natspec-docs`. Subjective take but I suspect many folks don't know what NatSpec is & may get confused.

Reply:
I wonder if there's also a way to specify the display name of the directory (e.g. to "Solidity docs" or "Solidity API") 🤔 

![Image 2023-12-11 at 21 44 46](https://github.com/privacy-scaling-explorations/maci/assets/8097623/cf43d797-5918-4e61-8c25-66e95b50ab43)

Reply:
I think we can accomplish that with a custom sidebar, can do that in a later PR if you want - we can directly specify order and name there

Reply:
> I think we can accomplish that with a custom sidebar, can do that in a later PR if you want - we can directly specify order and name there

Sweet, love that idea!